edenia/antelope-tools
# Feat/evm transactions and token history

### Add queries to get the EVM transactions and token incoming/outgoing history 

### What does this PR do?

- Add needed actions and routes to get the historical data
- Add guest permissions to EVM stats view

### Steps to test

1. Run the project locally with EVM backend
2. Check the queries evm_token_history and evm_transactions_history in the hasura console
3. Check the correctness of the queries.
